templates/sag-mft-config/template.yaml

```diff
@@ -21,22 +34,160 @@ templates:
           configuration:                        
             OSGI-IS_${instance.name}-WmMFT:
               COMMON-PORTS:
-                COMMON-PORTS-MFT${mft.port.name}_${mft.port.number}:
+                COMMON-PORTS-MFT${mft.ftp.port.name}_${mft.ftp.port.number}:
```

Comment:
The default value of the port name already contains the number 
Is the variable for teh port number needed in this case